doc/db-schema.md

```diff
@@ -0,0 +1,37 @@
+# Database Schema
+
+```
+                     Table "public.schema_version"
+     Column     |            Type             |       Modifiers
+----------------+-----------------------------+------------------------
+ installed_rank | integer                     | not null
+ version        | character varying(50)       |
+ description    | character varying(200)      | not null
+ type           | character varying(20)       | not null
+ script         | character varying(1000)     | not null
+ checksum       | integer                     |
+ installed_by   | character varying(100)      | not null
+ installed_on   | timestamp without time zone | not null default now()
+ execution_time | integer                     | not null
+ success        | boolean                     | not null
+Indexes:
+    "schema_version_pk" PRIMARY KEY, btree (installed_rank)
+    "schema_version_s_idx" btree (success)
+```
```

Comment:
is Flyway table relevant?

Reply:
I would have thought the flyway table is implementation detail of flyway, they could change the syntax and as long as it is seamless we would have no idea.

Reply:
yeh, fair comment. will remove